build-webCI.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Note that this Azure DevOps pipeline is largely redundant with /.github/workflows/ci.yml, which
+# is what we use instead for validation builds of pull requests. This CI build only runs against
+# pushes to main, not PRs; it exists in tandem with the GitHub Actions CI build because it
+# integrates better with our ADO-based release process.
+
+trigger:
+    branches:
+        include:
+            - main
+
+variables:
+    windowsImage: 'windows-2022-secure'
+    macImage: 'macOS-11'
+    linuxImage: 'ubuntu-22.04-secure'
+    Codeql.Enabled: true
+
+resources:
+    repositories:
+        - repository: 1esPipelines
+          type: git
+          name: 1ESPipelineTemplates/1ESPipelineTemplates
+          ref: refs/tags/release
+
+extends:
+    template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+    parameters:
+        pool:
+            name: $(a11yInsightsPool) # Name of your hosted pool
+            image: windows-2022-secure # Name of the image in your pool. If not specified, first image of the pool is used
+            os: windows
+
+        stages:
```

Comment:
@v-prachin it looks like lines 35-182 of this file are identical to lines 41-188 of build.yaml. That's an 80% overlap in terms of lines., which almost guarantees that they'll eventually get out of sync.  Is there a way that we can consolidate these 147 lines into a single yml file for maintenance purposes?

Reply:
Added a new yml accessibility-insights-web.yml for this suggestion.